tests/inlining_test.py

```diff
@@ -127,15 +127,16 @@ def outerprog(A: dace.float64[20]):
         nested(A)
 
     sdfg = outerprog.to_sdfg(simplify=True)
-    from dace.transformation.interstate import InlineMultistateSDFG
-    sdfg.apply_transformations(InlineMultistateSDFG)
-    assert sdfg.number_of_nodes() in (4, 5)
 
     A = np.random.rand(20)
     expected = np.copy(A)
-    outerprog.f(expected)
+    sdfg(expected)
```

Comment:
The test may fail in unexpected ways. `outerprog.f ` will run the original function and is what we expect should be the result of the function